### PR TITLE
New site code update

### PR DESCRIPTION
**Pull request by jDramaix**

### CODE REVIEW

**Comment by jDramaix** [comment]
@jasonlemay @meriouma  Could you guys sign the CLA ?

**Comment by meriouma** [comment]
Done

File: src/main/resources/main.tpl [hunk 1]
```diff
@@ -1,89 +1,122 @@
...
+
+        <nav role="navigation" id="nav" class="closed">
+            <div class="wrapper">
+                <ul>
+                    <li class="logo"><a href="/">GWT homepage</a></li>
+                    <li><a href="overview.html"><i class="icon_overview"></i>Overview</a></li>
+                    <li class="sep"><a href="gettingstarted.html"><i class="icon_getstarted"></i>Get started</a></li>
+                    <li><a href="doc/latest/tutorial/index.html" class="active"><i class="icon_tutorials"></i>Tutorials</a></li>
+                    <li><a href="doc/latest/DevGuide.html"><i class="icon_doc"></i>Docs</a></li>
+                    <li><a href="examples.html"><i class="icon_ressources"></i>Ressources</a></li>
```
Comment (spg) [inline src/main/resources/main.tpl]: Typo: `Resources`

Comment (spg) [inline src/main/resources/main.tpl]: See https://github.com/gwtproject/gwt-site/issues/23

**Comment by Pacane** [comment]
I've signed the CLA too!

**Comment by jDramaix** [comment]
wake you up googlebot :-)

**Comment by Pacane** [comment]
Hey @googlebot I confirm it's my own work and agree with merging my code in the gwt-site repository"

**Comment by jasonlemay** [comment]
It's okay @googlebot, let @jDramaix merge my code

**Comment by meriouma** [comment]
Hey @googlebot I confirm it's my own work and agree with merging my code in the gwt-site repository

**Comment by branflake2267** [comment]
Why all the .js stuff?

**Comment by branflake2267** [comment]
Not sure I'm a fan of the .js files. This looks like it adds complexity.

**Comment by tbroyer** [comment]
Too many unrelated things.

Remove IDE-specifics from gitignore (setup a global gitignore for that).

Not a fan at all of the Node dependency; where will the maven plugin install those things?. Is there a way to build with only markdown rendering?

**Comment by jDramaix** [comment]
Yep, the grunt (and so the node) dependency will be removed in the future. I didn't have time to do it before gwtcreate and I plan to work on that next month. I propose just to open an issue in order to track these and don't let them in the code base forever. 

@tbroyer  @branflake2267  Do you agree with that ?

Also as you are more expert than me with maven, do you know any good maven plugin that we can use in order to:
- compile less file to css during the build
- minify js file.

The last point is not really important because we use js only in order to display the mobile menu. I plan to replace that by pur gwt code.

**Comment by tbroyer** [comment]
Works for me.

**Comment by branflake2267** [comment]
Thats fine. I was surprised to see non gwtish stuff some plain old javascript imports. I'm often in the thought pattern we should dog food what the library does, so if we can't do it yet, haven't yet provided that feature I forego it until its offered, providing some drive to make that feature. I'm sure its fine, looks great adds something cool, although I had thought at first that it adds a mechanics which adds weight to the process. My goal with docs/guides at work is keep the application as light as possible, less is more, because writing the stuff is much more critical to me, but maybe that's because I'm slightly lazy in the fact I don't want to have to maintain all the layers all the time :). Well anyway, one things for sure, you and your cohorts in arms are making some nice effort so you get big bonus points for that. Nice job none the less.

**Comment by jDramaix** [comment]
The idea was to used only gwt code but I didn't have enough time before gwtcreate to replace the javascript stuff added by the web integrator.

Will be done later

**Comment by branflake2267** [comment]
yeah sounds good, sorry I got to rambling :)

**Pull request merged**